website/content/en/docs/main/tutorials/shared/cli-installation.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
drop frontmatter from these shared markdowns and using the built-in shortcode for file inclusion (instead of the new `include` shortcode you created that removes the text between `---`